crates/bevy_render/src/camera/camera.rs

```diff
@@ -823,6 +844,7 @@ pub struct ExtractedCamera {
     pub clear_color: ClearColorConfig,
     pub sorted_camera_index_for_target: usize,
     pub exposure: f32,
+    pub render_target_layer: Option<NonMaxU32>,
```

Comment:
Nbd, but this field wasn't immediately intuitive to me and part of me feels like it should be a direct property of `NormalizedRenderTarget::Image` or a new `NormalizedRenderTarget::Cubemap` instead.